tgstation.dme

```diff
@@ -15,6 +15,9 @@
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
 #include "_maps\tgstation2.dm"
+#include "_maps\barshuttle.dmm"

```

Comment:
I assume there's probably a better place to put the map files. But I needed to put them somewhere.